src/main/java/Gulie.java

```diff
@@ -0,0 +1,41 @@
+import java.io.InputStream;
+import java.io.PrintStream;
+
+public class Gulie {
+    private GulieTasklist tasklist;
+    private final static String savepath = "./data/Gulie.txt";
```

Comment:
Remember to follow the SCREAMING_SNAKE_CASE format for your constants 😁

```suggestion
    private final static String SAVE_PATH= "./data/Gulie.txt";
```